Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiaryModule.kt

```diff
@@ -0,0 +1,32 @@
+package de.rki.coronawarnapp.contactdiary.ui
+
+import dagger.Module
+import dagger.android.ContributesAndroidInjector
+import de.rki.coronawarnapp.contactdiary.ui.day.ContactDiaryDayFragment
+import de.rki.coronawarnapp.contactdiary.ui.day.ContactDiaryDayModule
+import de.rki.coronawarnapp.contactdiary.ui.day.sheets.location.ContactDiaryLocationBottomSheetDialogFragment
+import de.rki.coronawarnapp.contactdiary.ui.day.sheets.location.ContactDiaryLocationBottomSheetDialogModule
+import de.rki.coronawarnapp.contactdiary.ui.day.sheets.person.ContactDiaryPersonBottomSheetDialogFragment
+import de.rki.coronawarnapp.contactdiary.ui.day.sheets.person.ContactDiaryPersonBottomSheetDialogModule
+import de.rki.coronawarnapp.contactdiary.ui.day.tabs.location.ContactDiaryLocationListFragment
+import de.rki.coronawarnapp.contactdiary.ui.day.tabs.location.ContactDiaryLocationListModule
+import de.rki.coronawarnapp.contactdiary.ui.day.tabs.person.ContactDiaryPersonListFragment
+import de.rki.coronawarnapp.contactdiary.ui.day.tabs.person.ContactDiaryPersonListModule
+
+@Module
+abstract class ContactDiaryModule {
+    @ContributesAndroidInjector(modules = [ContactDiaryDayModule::class])
```

Comment:
Can you move this to your own module and just include it here? Where are the methods that inject the repository now?

Reply:
moved some stuff around 👍